On OpenBSD tcp_var.h also requires tcp_timer.h.

/usr/include/netinet/tcp_var.h:72:25: error: use of undeclared identifier 'TCPT_NTIMERS'
        struct timeout t_timer[TCPT_NTIMERS];   /* tcp timers */

Nothing from tcp_var is used here at the moment anyway so don't pull it in.

Index: internal.h
--- internal.h.orig
+++ internal.h
@@ -137,9 +137,6 @@ typedef unsigned short sa_family_t;
 #include <netinet/udp.h>
 #include <netinet/icmp6.h>
 #include <netinet/tcp.h>
-#ifdef HAVE_NETINET_TCP_VAR_H
-#include <netinet/tcp_var.h>
-#endif
 #include <netinet/if_ether.h>
 #include <arpa/inet.h>
 #include <unistd.h>
